devenv/dev-dashboards/main.libsonnet

```diff
@@ -0,0 +1,99 @@
+(import 'datasource-elasticsearch/elasticsearch_compare.json') +
```

Comment:
For future gdev dashboards that are created we will need to modify this file to include that file in order for it to be available on play.grafana correct?

Reply:
@nmarrs it's generated by:

```
make devenv-dashboards
```

alternatively, i can add a `gen-libsonnet` target to the makefile and make that a dependency of the build related targets